cypress/integration/use-cases/wizard.spec.js

```diff
@@ -57,5 +57,28 @@ describe('Using the wizard', () => {
             .should('include', '#!from=wizard');
 
         cy.get('.joyride-tooltip__close').click();
+
+        cy.get('aside.company-info.box');
+
+        cy.get('.dynamic-input.dynamic-input-name input[name=value]').type('Kim Mustermensch');
+        cy.get('.request-transport-medium-chooser')
+            .contains('Fax')
+            .click();
+
+        cy.contains('Download PDF')
+            .should('not.have.class', 'disabled')
+            .clickLinkWithoutFollowingHref();
+
+        cy.contains('Next request').click();
+        cy.get('aside.company-info.box').get('.company-info-content');
```

Comment:
Why do you do that here? The way I understand it, this should do nothing, right?

Reply:
Quite the opposite. If the company info box isn't there (which was one symptom of the bug), the test will fail. Almost all Cypress commands have implicit assertions built-in, makes it really easy to write tests.

Reply:
I wasn't aware of that. Good to know!